### PR TITLE
CI pipelines for docker images

### PR DESCRIPTION
**Pull request by gwthm-in**
Have to add master branch conditions as well. 
Will do once testing is done. 

Going to tag this commit and see if pipeline pushes docker images to hub.

### CODE REVIEW

**Comment by rShetty** [comment]
Merge errors

File: .travis.yml [hunk 1]
```diff
@@ -26,5 +29,14 @@ deploy:
     on:
       tags: true
 
+  - provider: script
+    secure: "c3DI8KXPuVlJ9AerXw0ZIFvQVK/ImNzryDnbZdeXnld657Y63h6yhR8Y92NSydahoaG7cvDEQK6hJDSDc8j/vB3LVvfAAn6ZfhfYKsp40wEBf7Q6eByfddZ7U4Yfc/dCoKBZaQHkohPCxM/DMTg9PaAD73+jqTfYydQUHcJx9hLoBIin8IkzyJ0O7Lf8ZPCBYitV2QwrJ0x9uwzezavy1xwtraCyjQ2QNF0IebKnSFXfKmagUj77wWkVVYGBGmjYlWZV1g0SISHsNbx1E4ftNHrraf2pMoMgSEUSyzEouklLxawoHokbf0fc9arDk0jg+1jhu/2waOXFYyAMidglwhBe0C6fs9qf+r/mDUJqB6M1a3pRAGswjysE4wIynQ3Vkqz8RYQ4dWaPamwuAQIizjO4XmilAh1kHn6qMwiDanJ+AJJcPK2LhnP7n6AWGtkjTzR7y3hqcJBqi3hVNsILkutG0LaR+Eu1cKO+vGFeBa5jIBSXnUCY+2kZrWTBaYEHbrmAfY+mISwaWh3YzNbHBoZHJt9QLLHokt1BdD/xOK7DHkKPta8YqHyUL7EavOw8e76TNtofOEntMlubb2dhUJ0PGawX6nGnoMGyV8JgFUIkUkiZydiU3of/Qvm9f32rAcHZQlyzuC7EDCmD113ndYczxzsd6VvOuRw4O7+VBv4="
```
Comment (rShetty) [inline .travis.yml]: What is this secure ?

Comment (irfn) [reply .travis.yml]: Check this [doc](https://docs.travis-ci.com/user/encryption-keys/) and this blog [secure-environment-variables-with-travis](https://brettdewoody.com/secure-environment-variables-with-travis/)
Dont particularly like this way but seems an accepted practice in travis ci

**Comment by gwthm-in** [comment]
Have to mark it as ready for review in order to trigger the BUILD pipeline and tag it. So this will be test build which pushes to docker hub. Once we verify, will add *only on master* and delete the from docker hub. @rShetty @irfn

**Comment by gwthm-in** [comment]
@rShetty Can not check this as travis won't run deploy stage on PRs. Please merge if it looks good to you.

**Pull request merged**